Automattic/wp-calypso
# Jetpack Connect: Introduce NUX Site User Type step.

Introduce a site user type NUX step. Here, we determine whether a given user is setting up a site for themselves or for someone else.

The step saves the data and is part of the flow: `site topic  -> user type -> plans `. 

<img width="482" alt="Screenshot 2019-03-14 at 10 03 46" src="https://user-images.githubusercontent.com/13561163/54344074-7ec1eb80-4640-11e9-8052-9991f948bb09.png">
new design:

<img width="551" alt="Screenshot 2019-03-21 at 01 28 33" src="https://user-images.githubusercontent.com/13561163/54727624-1b244a80-4b79-11e9-9dd6-04dc1f7c3cb4.png">

#### Testing instructions


- Spin up calypso.live or local branch.
- Go to /jetpack/user-type/:site for a connected Jetpack site with current JP master (e.g. `https://jurassic.ninja/create?jetpack-beta`)
- Make a selection and submit
- Verify that the option was saved correctly in `site_user_type` option in `/wp-admin/options.php`—
   MYSELF is saved as 'creator', and FOR OTHERS as 'builder'
- Test complete NUX flow, the screen should appear after the site vertical step. In order to access he flow, copy the url behind of the `Set up Jetpack` connection button and append `&calypso_env=development`



note: we will remove the JITM in the follow up PR https://github.com/Automattic/wp-calypso/pull/31447
